warehouse/query-core/src/main/java/datawave/query/postprocessing/tf/FieldValue.java

```diff
@@ -0,0 +1,131 @@
+package datawave.query.postprocessing.tf;
+
+import datawave.util.StringUtils;
+import org.apache.accumulo.core.data.Key;
+import org.apache.hadoop.io.Text;
+
+/**
+ * A field name and value which is sorted on {@code <value>\0<name>}
+ */
+@Deprecated
```

Comment:
Consider whether we should just delete this outright.

Reply:
If for some reason we need to flip back to this as opposed to the `TFKey` I would like to leave this in. We can always remove this later as part of a sweep for deprecated classes.